fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -353,25 +352,8 @@ private void replaceTableAlias(SlotRef slotRef,
         private void checkDistribution(CreateMaterializedViewStatement statement,
                                        Map<TableName, Table> tableNameTableMap) {
             DistributionDesc distributionDesc = statement.getDistributionDesc();
-            Map<String, String> properties = statement.getProperties();
             List<Column> mvColumnItems = statement.getMvColumnItems();
 
-            // For replication_num, we select the maximum value of all tables replication_num
```

Comment:
It is better to follow the user's setting when setting it. If the user does not set it, we simply speculate on this parameter.